parlai/tasks/self_feeding/agents.py

```diff
@@ -10,11 +10,9 @@
 
 import numpy as np
 
-from .build import build
-
-import parlai.core.agents as core_agents
-from parlai.core.teachers import ParlAIDialogTeacher
+from parlai.core.teachers import ParlAIDialogTeacher, MultiTaskTeacher
 from projects.self_feeding.utils import add_person_tokens
+from .build import build
```

Comment:
huh, I'm curious about this one.

Reply:
Do you think it's worth changing? A lot of agents do relative imports like that

Reply:
I mean i'm just a little surprised something was broken, i thought this was tested.

Reply:
Hmm, nothing in this agent was broken - the same things are imported before and after this PR, but I think my IDE shuffled around the order of the import lines when I moved `MultiTaskTeacher` into a different module